app/views/posts/index.html.erb

```diff
@@ -1 +1,38 @@
-<h1>See my posts</h1>
\ No newline at end of file
+<div class="main">
+  <div class="content">
+    <h1><%= @user.name %>'s Posts</h1>
+    <div class='box flex'>
+      <div class='img flex'>
+        <img src="<%= @user.photo %>" alt="unsplash">
+      </div>
+      <div class='user-details'>
+        <p><%= @user.name %></p>
+          <span class="post-no">number of posts: <%= @user.posts_counter %></span>
+      </div>
+    </div>
+
+    <% @posts.each do |post| %>
+      <div class="post">
+        <p><b><%= post.title %></b></p>
+        <p><%= post.text %></p>
+        <div class="come_like">
+          <span>comments: <%= post.comments_counter %>, likes: <%= post.likes_counter %></span>
+        </div>
+        <div class="comments">
+          <% if post.comments_counter == 0 %> 
+            <span>There are no comments for this post.</span>
+          <% else %>
+            <% post.recent_comments.each do |comment| %>
+              <p><%= comment.author.name %>: <%= comment.text %></p>
+            <% end %>
+            </div>
+              <button class="post-btn">
+                <%= link_to "See Post", user_post_path(post.author_id, post.id) %>
+              </button>
+            </div>
+          <% end %>
+        </div>
+      </div>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you use `partial` file for one `Post` card for best practices. I suggest you refer this **[article](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials)** to help you how to do it and more details about partail files.